tasks/translation/translate_content.py

```
@@ -0,0 +1,216 @@
+from typing import List
+from cognee.tasks.translation.models import TranslatedContent, LanguageMetadata
+# Import DocumentChunk from the appropriate module
+
+async def translate_content(
+    data_chunks: List['DocumentChunk'],
+    target_language: str = "en",
+    translation_provider: str = "openai",
```

Ruff (0.12.2)

Unused function argument: `translation_provider`

(ARG001)